### DISCUSSION TITLE
Strongswan offline install?

### DISCUSSION THREAD

**Discussion by davehouser1** [strongswan/strongswan#688] [Q&A] [answered]
Where can I download the strongswan rpm files to upload to an offline server?
The box I am installing on has no connection to the internet.
Rather not have to compile from source. Any recommendations?

**Comment by tobiasbrunner** [comment, marked as answer]
See [InstallationDocumentation](https://wiki.strongswan.org/projects/strongswan/wiki/InstallationDocumentation#Distribution-packages) for possible binary package sources.

**Reply by davehouser1** [reply]
Thanks, I found the .rpm file I needed and was able to deploy.
